frontend/lib/src/components/widgets/FileUploader/FileUploader.tsx

```diff
@@ -15,7 +15,7 @@
  */
 
 import axios from "axios"
-import _ from "lodash"
+import isEqual from "lodash/isEqual"
```

Comment:
```suggestion
import isEqual from "lodash/isEqual"
import zip from "lodash/zip"
```

Reply:
This line needs to be updated:
https://github.com/streamlit/streamlit/pull/7842/files#diff-55a99c6849d60c1b1045ab60477d4c719e95fd629df5d92c07b888c36bba6906R250